src/screens/FeedScreen/Header.js

```diff
@@ -6,7 +6,15 @@ import * as React from 'react';
 import Image from 'react-native-fast-image';
 import PropsTypes from 'prop-types';
 import moment from 'moment';
-import {Dimensions, Platform, SafeAreaView, StyleSheet, Text, View} from 'react-native';
+import {
+  Dimensions,
+  Platform,
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View
+} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been restructured for better readability and maintainability. This is a good practice as it makes the code cleaner and easier to understand.
```diff
- import {Dimensions, Platform, SafeAreaView, StyleSheet, Text, View} from 'react-native';
+ import {
+   Dimensions,
+   Platform,
+   SafeAreaView,
+   StyleSheet,
+   Text,
+   TouchableOpacity,
+   View
+ } from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->